jd4/integration_test.py

```diff
@@ -80,7 +80,7 @@ def test_php(self):
 """)
 
     def test_py(self):
-        self.do_lang('py', b'print sum(map(int, raw_input().split()))')
+        self.do_lang('py', b'print(sum(map(int, input().split())))')
```

Comment:
why not just drop `py`?

Reply:
To maintain the compatibility with the front end...